Engine/Component/ComponentParticleSystem.cpp

```diff
@@ -29,6 +29,7 @@ void ComponentParticleSystem::Init()
 
 	billboard = new ComponentBillboard(this->owner);
 	billboard->ChangeBillboardType(ComponentBillboard::AlignmentType::WORLD);
+	billboard->ChangeTexture(3665414218); //default particle texture
```

Comment:
Put this number in a const variable, this way is more readable and you wont need to put the comment